packages/scanner-global-library/src/setup-scanner-container.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as inversify from 'inversify';
+import { cloudAxeConfiguration, localAxeConfiguration } from './factories/axe-configuration';
+import { AxePuppeteerFactory } from './factories/axe-puppeteer-factory';
+import { iocTypes } from './ioc-types';
+
+export function setupCloudScannerContainer(container: inversify.Container): inversify.Container {
+    container.bind(iocTypes.AxeConfiguration).toConstantValue(cloudAxeConfiguration);
+    container.bind(AxePuppeteerFactory).toSelf().inSingletonScope();
+
+    return container;
+}
+
+export function setupLocalScannerContainer(container: inversify.Container): inversify.Container {
+    container.bind(iocTypes.AxeConfiguration).toConstantValue(localAxeConfiguration);
```

Comment:
Do we need to bind AxePuppeteerFactory here as well?

Reply:
I don't think it makes any practical difference whether local uses a singleton or an instance-per-scan; I left that part of the logic as-is only because there didn't seem to be any compelling reason to change it.